system-variables.md

```diff
@@ -4782,6 +4782,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 这个变量用于设置 window 算子的并行度。
 - 默认值 `-1` 表示使用 `tidb_executor_concurrency` 的值。
 
+### `tidb_request_source_type` <span class="version-mark">从 v7.4.0 版本开始引入</span>
```

Comment:
@nolouch Please add `tidb_request_source_type` to release-7.4.0.md:
https://github.com/pingcap/docs-cn/blob/a9062ca0c6401a9674f074157c055505ce905e13/releases/release-7.4.0.md?plain=1#L289-L304

Reply:
Done